apps/browser-extension-wallet/src/features/address-book/components/AddressDetailDrawer/AddressChangeDetailDrawer.tsx

```diff
@@ -0,0 +1,275 @@
+    // eslint-disable-next-line unicorn/no-null
+    setSelectedSelectedId(null);
+    if (action === 'delete') {
+      sendAnalytics(
+        popupView
+          ? AnalyticsEventNames.AddressBook.CONFIRM_DELETE_UPDATE_ADDRESS_POPUP
+          : AnalyticsEventNames.AddressBook.CONFIRM_DELETE_UPDATE_ADDRESS_BROWSER
+      );
+      onDelete(selectedId);
+      onCancelClick();
+    } else {
+      sendAnalytics(
+        popupView
+          ? AnalyticsEventNames.AddressBook.CONFIRM_UPDATE_ADDRESS_POPUP
+          : AnalyticsEventNames.AddressBook.CONFIRM_UPDATE_ADDRESS_BROWSER
+      );
+      onConfirmClick(initialValues);
+      onCancelClick();
+    }
+  };
+
+  return (
+    <>
+      <Drawer
+        keyboard={false}
+        zIndex={999}
+        onClose={onCancelClick}
+        className={cn(styles.drawer, { [styles.popupView]: popupView })}
+        title={<DrawerHeader title={t('addressBook.reviewModal.title', { name: initialValues.name })} />}
+        open={visible}
+        navigation={
+          <DrawerNavigation
+            onCloseIconClick={!popupView ? onCancelClick : undefined}
+            onArrowIconClick={popupView ? onCancelClick : undefined}
+          />
+        }
+        footer={
+          <div className={styles.footer}>
+            <Button
+              data-testid="address-form-details-btn-update"
+              onClick={() => {
+                sendAnalytics(
+                  popupView
+                    ? AnalyticsEventNames.AddressBook.UPDATE_ADDRESS_POPUP
+                    : AnalyticsEventNames.AddressBook.UPDATE_ADDRESS_BROWSER
+                );
+                setSelectedSelectedId(initialValues.id);
+                setAction(ACTIONS.UPDATE);
+              }}
+              size="large"
+              block
+            >
+              {t('addressBook.reviewModal.confirmUpdate.button')}
+            </Button>
+            <Button
+              data-testid="address-form-details-btn-delete"
+              onClick={() => {
+                sendAnalytics(
+                  popupView
+                    ? AnalyticsEventNames.AddressBook.DELETE_UPDATE_ADDRESS_PROMPT_POPUP
+                    : AnalyticsEventNames.AddressBook.DELETE_UPDATE_ADDRESS_PROMPT_BROWSER
+                );
+                setSelectedSelectedId(initialValues.id);
+                setAction(ACTIONS.DELETE);
+              }}
+              color="secondary"
+              size="large"
+              className={styles.deleteButton}
+              block
+            >
+              {t('addressBook.reviewModal.cancelUpdate.button')}
+            </Button>
+          </div>
+        }
+        popupView={popupView}
+      >
+        <>
+          <div className={styles.warningBanner}>
+            <Banner withIcon message={t('addressBook.reviewModal.banner.description', { name: initialValues.name })} />
+          </div>
+          <div className={styles.addressContainer}>
+            <Timeline>
```

Comment:
Try this for the white background.

```suggestion
            <Timeline className={styles.sideTimeline}>
```

Add this to the CSS file
```
.sideTimeline {
      margin: size_unit(7) 0 size_unit(7) size_unit(7.75);
      :global {
        .ant-timeline-item {
          height: 130px;
        }
        .ant-timeline-item-content {
          @include text-bodySmall-medium;
          color: var(--text-color-primary);
          margin-left: 17px;
        }
        .ant-timeline-item-head {
          width: size_unit(2.5);
          height: size_unit(2.5);
          background: var(--bg-color-container);
        }
        .ant-timeline-item-tail {
          left: 0;
          border-left: 2px solid var(--light-mode-light-grey-plus, var(--dark-mode-mid-grey));
        }
      }
    }
```

Reply:
<img width="346" alt="Screenshot 2023-07-14 at 15 18 58" src="https://github.com/input-output-hk/lace/assets/4052063/4ffbf40f-0d43-4593-af4f-df5016beac15">

Reply:
b550f34579c8d4ede3e04728de44ad2e23664cce